src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1266,26 +1326,31 @@ export class NetworkAreaDiagramViewer {
         }
     }
 
-    private updateTextNodeMetadataCallCallback(mousePosition: Point) {
-        if (this.onMoveTextNodeCallback != null) {
-            // get from metadata node connected to moved text node
-            const node: NodeMetadata | undefined = this.diagramMetadata?.nodes.find(
-                (node) => node.svgId == DiagramUtils.getVoltageLevelNodeId(this.draggedElement?.id)
-            );
-            const textNode: TextNodeMetadata | undefined = this.diagramMetadata?.textNodes.find(
-                (textNode) => textNode.svgId == this.draggedElement?.id
-            );
-            if (node != null && textNode != null) {
-                // get new text node position
-                const textPosition = DiagramUtils.getTextNodeAngleFromCentre(this.draggedElement, mousePosition);
-                const textNodeMoves = DiagramUtils.getTextNodeMoves(textNode, node, textPosition, this.endTextEdge);
-                // update text node position in metadata
-                textNode.shiftX = textNodeMoves[0].xNew;
-                textNode.shiftY = textNodeMoves[0].yNew;
-                textNode.connectionShiftX = textNodeMoves[1].xNew;
-                textNode.connectionShiftY = textNodeMoves[1].yNew;
-                // call the node move callback, if defined
-                this.onMoveTextNodeCallback(
+    private updateTextNodeMetadataCallback(
+        vlNode: SVGGraphicsElement,
```

Comment:
Can this parameter be renamed for suggesting this is a text node, not a voltage level node?

Reply:
Done